src/Generator.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Generator.hpp"
+#include <iostream>
```

Comment:
iostream is not used here, please remove this include

Reply:
will be removed, this is relic from testing phase